src/RestrictedPython/Guards.py

```diff
@@ -15,6 +15,7 @@
 # AccessControl.ZopeGuards contains a large set of wrappers for builtins.
 # DocumentTemplate.DT_UTil contains a few.
 
+from . import _compat
```

Comment:
sorry but I do not like local imports, please use absolute imports.

Reply:
Fixed.